app/partials/app.cjsx

```diff
@@ -11,7 +11,8 @@ class GeordiLogger # Make calls to the Geordi API to log user activity
 
   @tokens = ['zooHome', 'zooTalk', 'zooniverse/gravity-spy']
 
-  keys: {}
+  keys: 
+    projectToken: 'zooHome'
```

Comment:
I don't think the defaulting to zooHome should be part of geordi-client. That is PFE-specific and should be passed in. Remember the client should be usable on non-PFE sites.